lib/msal-browser/src/index.ts

```diff
@@ -103,3 +84,9 @@ export {
 } from "@azure/msal-common";
 
 export { version } from "./packageMetadata";
+
+/**
+ * Warning: This set of exports is purely intended to be used by other MSAL libraries, and should be considered potentially unstable and we strongly recommend against relying on them directly, and instead using the main export.
```

Comment:
Rephrasing to avoid the run on sentence
```suggestion
 * Warning: This set of exports is purely intended to be used by other MSAL libraries, and should be considered potentially unstable. We strongly discourage using them directly, you do so at your own risk.
```